templates/docs.hamlet

```diff
@@ -0,0 +1,121 @@
+<h1>Carnival Documentation
+
```

Comment:
Can we add an introduction? What's Carnival? How does it work, at a high-level? Let's make a page that people can read without looking at the README.